src/components/album/AlbumChat.js

```diff
@@ -10,15 +11,19 @@ const AlbumChat = ({ apiUrl }) => {
   const albumId = match ? match[1] : null;
 
   const { user: loggedInUser } = useAuth();
-  const userId = loggedInUser?.user?.id;
+  const user = loggedInUser?.user?.id;
+
+  const [messages, setMessages] = useState([]);
+  const messagesContainerRef = useRef(null);
 
   // Handle sending a chat message
   const handleSendMessage = () => {
     if (socket && albumId) {
       const messageData = {
         message,
         albumId,
-        userId,
+        user,
+        isOwnMessage: true,
```

Comment:
Currently, when userA **_sends_** a message, they're sending it to the socket with `isOwnMessage: true`. And when userB (and all the other users in the same chat room) _**receives**_ that message on their browser, it will still say `isOwnMessage: true` because the backend was told to just forward along the `data` it was given. 

A different approach I would suggest, is _**not**_ including `isOwnMessage` in the message we send to the backend. Instead, when we receive the message in the frontend (in the `chat:album` listener on line 56) we can compare the id of the user who sent that message, to the logged-in user id that this component has on line 14 from the `useAuth` hook and set the `isOwnMessage` value there.

Here's some pseudocode to illustrate my suggestion:

```
newSocket.on('chat:album', (data) => {
   ...existing code...

  const isOwnMessage = loggedInUserId === data.user._id;

  setMessages((prevMessages => [...prevMessages,  { ...data, isOwnMessage } ]);
}
```